--- procmail-3.22/src/formisc.h.orig	2010-01-10 22:53:39.726707771 +0100
+++ procmail-3.22/src/formisc.h	2010-01-10 22:54:57.934199882 +0100
@@ -17,4 +17,4 @@ void
 char*
  skipwords P((char*start));
 int
- getline P((void));
+ pm_getline P((void));
--- procmail-3.22/src/fields.c.orig	2010-01-10 22:55:07.463366037 +0100
+++ procmail-3.22/src/fields.c	2010-01-10 22:55:28.953363478 +0100
@@ -110,16 +110,16 @@ void dispfield(p)register const struct f
 		    /* try and append one valid field to rdheader from stdin */
 int readhead P((void))
 { int idlen;
-  getline();
+  pm_getline();
   if((idlen=breakfield(buf,buffilled))<=0) /* not the start of a valid field */
      return 0;
   if(idlen==STRLEN(FROM)&&eqFrom_(buf))			/* it's a From_ line */
    { if(rdheader)
 	return 0;			       /* the From_ line was a fake! */
-     for(;buflast=='>';getline());	    /* gather continued >From_ lines */
+     for(;buflast=='>';pm_getline());	    /* gather continued >From_ lines */
    }
   else
-     for(;;getline())		      /* get the rest of the continued field */
+     for(;;pm_getline())		      /* get the rest of the continued field */
       { switch(buflast)			     /* will this line be continued? */
 	 { case ' ':case '\t':				  /* yep, it sure is */
 	      continue;
--- procmail-3.22/src/formisc.c.orig	2010-01-10 22:55:37.473363034 +0100
+++ procmail-3.22/src/formisc.c	2010-01-10 22:55:45.763361970 +0100
@@ -115,7 +115,7 @@ void loadchar(c)const int c;		      /* a
   buf[buffilled++]=c;
 }
 
-int getline P((void))			   /* read a newline-terminated line */
+int pm_getline P((void))			   /* read a newline-terminated line */
 { if(buflast==EOF)			 /* at the end of our Latin already? */
    { loadchar('\n');					  /* fake empty line */
      return EOF;					  /* spread the word */
