diff -ruNp procmail-3.24.orig/src/autoconf procmail-3.24/src/autoconf
--- procmail-3.24.orig/src/autoconf	2022-03-02 10:52:56.000000000 +0100
+++ procmail-3.24/src/autoconf	2025-03-02 07:41:27.305507604 +0100
@@ -90,7 +90,7 @@
 #Ok	#define USElockf
 #Ok	#define USEflock
 #Ok	#define MAILSPOOLDIR "/var/mail/"
-#Ok	#define SENDMAIL "/usr/sbin/sendmail"
+#Ok	#define SENDMAIL "/usr/lib/sendmail"
 #	#define buggy_SENDMAIL
 #	#define DEFflagsendmail ""
 #Ok	#define CF_no_procmail_yet
@@ -1416,49 +1416,11 @@ _autotst >>$ACONF
 
 found=no
 
-for a in /var/spool/mail /usr/spool/mail /var/mail /usr/mail /spool/mail
-do
-  test -d $a -a $found = no && found=$a
-done
-if test $found = no
-then
- found=/var/spool/mail
- echo Could not find the system-mailbox directory, supplied default.
-fi
-echo '#ifndef MAILSPOOLDIR' >>$ACONF
-echo '#define MAILSPOOLDIR "'$found'/"' >>$ACONF
-echo '#endif' >>$ACONF
-
-cat >lookfor <<HERE
-for a in /usr/sbin /usr/lib /lib /usr/etc /etc /usr/bin /bin /usr/local/bin \
- /usr/lbin /usr/local/lib /usr/local /usr/.lib /local/bin /sbin
-do
- if test -f "\$a/\$1"
- then
-    echo '#define SENDMAIL "'"\$a/\$1"'"' >>$ACONF
-    exit 0
- fi
-done
-exit 1
-HERE
-chmod 0755 lookfor
+echo '#define MAILSPOOLDIR "'/var/mail'/"' >>$ACONF && found=yes
 
-if _autotst 2 3 >>$ACONF
-   test $? != 0 ||
-    lookfor sendmail || lookfor smail || lookfor mail #|| lookfor rmail
-then
-:
-else
- echo 'Could not find any mailer.  It should be a mailer accepting at least'
- echo 'one plain destination address as its only argument (any sendmail'
- echo 'compatible mailer will do), and the mail-to-be-sent on stdin.'
- echo 'What is your mailer called?  (You can override this in config.h)'
- read a
- echo "#define SENDMAIL \"$a\"" >>$ACONF
-fi
+echo '#define SENDMAIL "/usr/lib/sendmail"' >>$ACONF
 
-grep '^#define SENDMAIL ".*/sendmail"' $ACONF >$DEVNULL ||
- echo "#define DEFflagsendmail \"\"" >>$ACONF
+echo "#define DEFflagsendmail \"\"" >>$ACONF
 
 a=/tmp/_chowntst.$$
 $RM -r $a
