diff -urN procmail-3.22/Makefile procmail-3.22.new/Makefile
--- procmail-3.22/Makefile	Tue Sep 11 06:53:09 2001
+++ procmail-3.22.new/Makefile	Sat Sep 15 18:18:47 2001
@@ -55,7 +55,7 @@
 
 LOCKINGTEST=__defaults__
 
-#LOCKINGTEST=/tmp .	# Uncomment and add any directories you see fit.
+LOCKINGTEST=/tmp .  # Uncomment and add any directories you see fit.
 #			If LOCKINGTEST is defined, autoconf will NOT
 #			prompt you to enter additional directories.
 #			See INSTALL for more information about the
diff -urN procmail-3.22/config.h procmail-3.22.new/config.h
--- procmail-3.22/config.h	Tue Sep 11 06:53:50 2001
+++ procmail-3.22.new/config.h	Sat Sep 15 18:19:23 2001
@@ -34,7 +34,7 @@
  * DEFSPATH and DEFPATH defines below
  */
 /*#define DEFSPATH	"PATH=/bin:/usr/bin"			/* */
-/*#define DEFPATH	"PATH=$HOME/bin:/bin:/usr/bin"		/* */
+/*#define DEFPATH	"PATH=$HOME/bin:/bin:/usr/bin:/sbin:/usr/sbin"	/* */
 
 /* every environment variable appearing in PRESTENV will be set or wiped
  * out of the environment (variables without an '=' sign will be thrown
@@ -46,7 +46,7 @@
  */
 #define PRESTENV	{"IFS","ENV","PWD",0}
 
-/*#define GROUP_PER_USER			/* uncomment this if each
+#define GROUP_PER_USER			/* uncomment this if each
 						   user has his or her own
 	group and procmail can therefore trust a $HOME/.procmailrc that
 	is group writable or contained in a group writable home directory
@@ -91,14 +91,14 @@
 	restriction does not apply to the /etc/procmailrc and
 	/etc/procmailrcs files) */
 
-/*#define NO_NFS_ATIME_HACK	/* uncomment if you're definitely not using
+#define NO_NFS_ATIME_HACK	/* uncomment if you're definitely not using
 				   NFS mounted filesystems and can't afford
 	procmail to sleep for 1 sec. before writing to an empty regular
 	mailbox.  This lets programs correctly judge whether there is unread
 	mail present.  procmail automatically suppresses this when it isn't
 	needed or under heavy load. */
 
-/*#define DEFsendmail	"/usr/sbin/sendmail"	/* uncomment and/or change if
+#define DEFsendmail	"/usr/lib/sendmail"	/* uncomment and/or change if
 						   the autoconfigured default
 	SENDMAIL is not suitable.  This program should quack like a sendmail:
 	it should accept the -oi flag (to tell it to _not_ treat a line
