diff -Nru procmail-3.15/Makefile procmail-3.15.new/Makefile
--- procmail-3.15/Makefile	Mon Sep 18 10:06:22 2000
+++ procmail-3.15.new/Makefile	Mon Sep 18 10:04:56 2000
@@ -55,7 +55,7 @@
 
 LOCKINGTEST=__defaults__
 
-#LOCKINGTEST=/tmp .	# Uncomment and add any directories you see fit.
+LOCKINGTEST=/tmp .  # Uncomment and add any directories you see fit.
 #			If LOCKINGTEST is defined, autoconf will NOT
 #			prompt you to enter additional directories.
 #			See INSTALL for more information about the
diff -Nru procmail-3.15/config.h procmail-3.15.new/config.h
--- procmail-3.15/config.h	Mon Sep 18 10:06:23 2000
+++ procmail-3.15.new/config.h	Mon Sep 18 10:07:32 2000
@@ -34,7 +34,7 @@
  * DEFSPATH and DEFPATH defines below
  */
 /*#define DEFSPATH	"PATH=/bin:/usr/bin"			/* */
-/*#define DEFPATH	"PATH=$HOME/bin:/bin:/usr/bin"		/* */
+/*#define DEFPATH	"PATH=$HOME/bin:/bin:/usr/bin:/sbin:/usr/sbin"	/* */
 
 /* every environment variable appearing in PRESTENV will be set or wiped
  * out of the environment (variables without an '=' sign will be thrown
@@ -46,7 +46,7 @@
  */
 #define PRESTENV	{"IFS","ENV","PWD",0}
 
-/*#define GROUP_PER_USER			/* uncomment this if each
+#define GROUP_PER_USER			/* uncomment this if each
 						   user has his or her own
 	group and procmail can therefore trust a $HOME/.procmailrc that
 	is group writable or contained in a group writable home directory
@@ -75,7 +75,7 @@
 	restriction does not apply to /etc/procmailrc and /etc/procmailrcs
 	files) */
 
-/*#define NO_NFS_ATIME_HACK	/* uncomment if you're definitely not using
+#define NO_NFS_ATIME_HACK	/* uncomment if you're definitely not using
 				   NFS mounted filesystems and can't afford
 	procmail to sleep for 1 sec. before writing a regular mailbox
 	(under heavy load procmail automatically suppresses this) */
@@ -86,7 +86,7 @@
    (either directly, or through changing the definitions in the same file
    of MAILSPOOLDIR, MAILSPOOLSUFFIX, MAILSPOOLHASH or MAILSPOOLHOME) */
 
-/*#define DEFsendmail	"/bin/mail"	/* uncomment and/or change if the
+#define DEFsendmail	"/usr/lib/sendmail"	/* uncomment and/or change if the
 					   autoconfigured default SENDMAIL is
 	not suitable */
 
